Sources/TuistDependencies/Carthage/CarthageInteractor.swift

```diff
@@ -0,0 +1,133 @@
+import RxBlocking
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+// MARK: - Carthage Interactor Errors
+
+enum CarthageInteractorError: FatalError, Equatable {
+    /// Thrown when Carthage cannot be found.
+    case carthageNotFound
+    /// Thrown when Carfile cannont be found in temporary directory after Carthage installation.
+    case cartfileNotFound
+
+    /// Error type.
+    var type: ErrorType {
+        switch self {
+        case .carthageNotFound, .cartfileNotFound:
+            return .abort
+        }
+    }
+
+    /// Error description.
+    var description: String {
+        switch self {
+        case .carthageNotFound:
+            return "Carthage was not found either in Bundler nor in the environment."
```

Comment:
Bundler is a tool to distribute Ruby Gems :). I'd rather say something like:

```
Carthage was not found in the environment. It's possible that the tool is not installed or hasn't been exposed to your environment.
``

Reply:
Of course. Sorry. Copy-paste issue. Im fixing it.